mergify_cli/ci_issues/cli.py

```diff
@@ -0,0 +1,82 @@
+import click
+
+from mergify_cli import utils
+from mergify_cli.ci_issues import junit_upload as junit_upload_mod
+
+
+ci_issues = click.Group(
+    "ci-issues",
+    help="Interact with Mergify's CI Issues",
+)
+
+
+@ci_issues.command(help="Upload JUnit XML reports")
+@click.option(
+    "--api-url",
+    "-u",
+    help="URL of the Mergify API",
+    required=True,
+    envvar="MERGIFY_API_SERVER",
+    default="https://api.mergify.com/v1",
+    show_default=True,
+)
+@click.option(
+    "--token",
+    "-t",
+    help="CI Issues Application Key",
+    required=True,
+    envvar="MERGIFY_CI_ISSUES_TOKEN",
```

Comment:
I think the token is global right now, so should we use something like `MERGIFY_TOKEN`? cc @sileht

Reply:
Currently you can't use the same token for Junit Upload and for the rest of the API.
At some point, we will be able to, so yes.